cli/tests/pcluster/example_configs/awsbatch.full.yaml

```diff
@@ -36,6 +36,7 @@ HeadNode:
       DeleteOnTermination: true
     EphemeralVolume:
       MountDir: /scratch
+  InternalSharedStorageType: ebs  # efs
```

Comment:
I'd say we should use the same name convention we're using in the storage type: `StorageType: Ebs`

Reply:
Ah ok that’s interesting because the enums we have use all lowercase

Reply:
Yup, I'm planning to convert it to lower case in the cluster_config, so it should be lower case everywhere else